global/scripts/debug.shlib

```diff
@@ -0,0 +1,262 @@
+#!/bin/echo This script should not run directly:
+
+# Description:
+#
+#   Function for printing debugging information on caught errors.
+#   It makes use of error trapping. To reset the trap, add
+#   end of the script using "trap - ERR".
+#
+# Input:
+#
+#   - line number
+#   - command ran
+#   - exit status
+
+trap 'debug_on_error $LINENO "$BASH_COMMAND" $?' ERR
+
+if [[ "$(type -t log_Err)" != "function" ]]
+then
+    source "$HCPPIPEDIR/global/scripts/log.shlib" "$@"
+fi
+
+
+# Description:
+#
+#   Function to turn on error trappimg
+
+debug_enable_trap()
+{
+    HCP_DEBUG_USETRAP=ON
+}
+
+# Description:
+#
+#   Function to turn on off error trappimg
```

Comment:
Typo

Reply:
To be clear, there is both "on off" and "trappimg" to be fixed here.

Reply:
corrected